SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -137,6 +137,13 @@ class PersonController: UIViewController {
         return button
     }()
     
+    lazy var viewBlur: UIVisualEffectView = {
+        let blurEffect = UIBlurEffect(style: .systemChromeMaterial)
```

Comment:
Is this to act as like a sort of loading view while the data is fetched?

Reply:
Yes it is